access_dynamo_from_python.py

```diff
@@ -0,0 +1,68 @@
+import boto3
+from boto3.dynamodb.conditions import Key
+from curses import is_term_resized
+import json
+import mysql.connector
+
+'''
+Function for retrieving an attribute from a RDS database that is used as
+foreign key in another DynamoDb database, and then retrieving the item 
+from the DynamoDb database.
+
+This could be done using the event lambda_handler receives, I did it this
+way instead, in order to show you the keys and parameters used.
+'''
+
+def lambda_handler(event, context):
+    event_for_rds = create_event_for_rds()
+    id = retrieve_foreign_key_from_rds_for_dynamo(event_for_rds)
+    event_for_dynamo = create_event_for_dynamo()
+    item = retrieve_item_from_dynamo_table(event_for_dynamo)
+    return item
+
+def retrieve_item_from_dynamo_table(event):
```

Comment:
this function should receive:
- table_name
- id

the event-related logic should be handled by the handler

the region should be a variable of the overall lambda